driver-core/src/main/com/mongodb/client/model/expressions/ExpressionHelper.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model.expressions;
+
+import org.bson.BsonArray;
+import org.bson.codecs.configuration.CodecRegistry;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+interface ExpressionHelper {
```

Comment:
The new way of writing helper classes ;)

Reply:
For my edification, why prefer using an interface over a class that can't be constructed?

Reply:
It's just a bit cleaner since you don't have to take extra steps to ensure non-instantiability (with final) or non-constructability (with a private constructor)